src/pydisagg/models.py

```diff
@@ -50,7 +52,9 @@ def __init__(
         m: float,
```

Comment:
Here let's remove the `,` after `m: float` and ruff will automatically format it into a single line. Currently it seems excessive to let a single parameter `m` occupy the entire line.